Update for `BackgroundFetchEvent` & `BackgroundFetchUpdateUIEvent` (#32163)
mdn/content: files/en-us/web/api/backgroundfetchevent/registration/index.md

## Examples

> In this example, if the user clicks on the user interface displaying the download progress, this fires the `backgroundfetchclick` event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.

Reviewer: ```suggestion
In this example, if the user clicks on the user interface displaying the download progress, this fires the [`backgroundfetchclick` event](/en-US/docs/Web/API/ServiceWorkerGlobalScope/backgroundfetchclick_event). The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.
```

> In this example, if the user clicks on the user interface displaying the download progress, this fires the `backgroundfetchclick` event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.

Author: ```suggestion
In this example, if the user clicks on the user interface displaying the download progress, this fires the {{domxref("ServiceWorkerGlobalScope/backgroundfetchabort_event", "backgroundfetchabort")}} event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.
```

> In this example, if the user clicks on the user interface displaying the download progress, this fires the {{domxref("ServiceWorkerGlobalScope/backgroundfetchabort_event", "backgroundfetchabort")}} event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.

Author: ```suggestion
In this example, if the user clicks on the user interface displaying the download progress, this fires the {{domxref("ServiceWorkerGlobalScope/backgroundfetchclick_event", "backgroundfetchclick")}} event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.
```
